locales/en.json

```diff
@@ -2229,13 +2228,11 @@
-				"advantages": "<li>Time and place-independent access to a large range of digital learning content</li><li>Exchange of digital, self-created materials</li><li>Digital education beyond separate computer rooms</li><li>Professional maintenance</li><li>Constant availability of the latest software for teachers and students</li><li>IT applications comply with the latest security standards</li><li>Contact point for new digital learning opportunities</li><li>Direct feedback from users</li><li>Suggestion for self-determined learning</li><li>Provision of specially designed learning offers</li><li>Networking of different learning locations</li>",
+				"advantages": "<li>Time and place-independent access to a large range of digital learning content</li><li>Exchange of digital, self-created materials</li><li>Digital education beyond separate computer rooms</li><li>Professional maintenance</li><li>Constant availability of the latest software for teachers and students</li><li>IT applications comply with the latest security standards</li><li>Contact point for new digital learning opportunities</li><li>Direct feedback from users</li><li>Suggestion for self-determined learning</li><li>Provision of specially designed learning offers</li><li>Networking of different learning locations</li><br>",
 				"furtherInformation": "Further information and help can be found <a href=\"https://bildungsserver.berlin-brandenburg.de/schul-cloud-einstieg\" target=\"_blank\" rel=\"noopener\">here <i class=\"fa fa-external-link\" aria-hidden=\"true\"></i></a>.",
-				"registration": "A uniform digital platform for the whole school offers many advantages. Registration for the {{shortTitle}} is therefore carried out by the school management. Please send an e-mail to the <a href=\"mailto:schul-cloud@bildungsserver.berlin-brandenburg.de\">Bildungsserver Berlin-Brandenburg</a>, if you want to register your school. \nAll students will then receive a registration link or login data from their school for the initial registration in the {{shortTitle}}. If you have any questions about registering for the first time, please contact the technical contact person of your school or the respective teacher.",
+				"registration": "The registration of the school for the {{shortTitle}} is carried out by the school management. The individual steps and helpful documents can be found on the <a href=\"https://bildungsserver.berlin-brandenburg.de/schul-cloud-einstieg\" target=\"_blank\" rel=\"noopener\"><span style='white-space: nowrap;'>Bildungsserver Berlin-Brandenburg <i class=\"fa fa-external-link\" aria-hidden=\"true\"></i></span></a>.<br>Teachers and students receive a registration link or login data from their school for the initial registration in the {{shortTitle}}.<br>If you have any questions about the registration process, teachers, students and parents should contact the designated School Cloud Admins at their school or the responsible teachers on site.",				
```

Comment:
Der letzte Part "If you have any questions about the registration process, teachers, students and parents should contact the designated School Cloud Admins at their school or the responsible teachers on site." klingt irgendwie komisch wenn man mit dem "If you" am Anfang. So ungefähr "wenn Sie fragen haben, dann fragen Lehrer..." und so weiter.

Reply:
Ist angepasst.